exporter/awsemfexporter/emf_exporter_test.go

```diff
@@ -46,7 +46,16 @@ func (p *mockPusher) ForceFlush() error {
 	return nil
 }
 
+// set fake environment variables for testing
+func setupTestEnv(t *testing.T) {
+	t.Helper()
+	t.Setenv("AWS_REGION", "us-west-2")
+	t.Setenv("AWS_ACCESS_KEY_ID", "fake")
+	t.Setenv("AWS_SECRET_ACCESS_KEY", "fake")
```

Comment:
I wonder if this is going to set off automated security scanners for a "presence of credentials". 🤔

Reply:
I think we can add exceptions if they complain, but happy to look into a different implementation if you have suggestions